### PR TITLE
gnome-terminal: add pt_BR translation

### PR DESCRIPTION
**Pull request by T16K**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pt_BR/linux/gnome-terminal.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# gnome-terminal
+
+> O emulador do Terminal GNOME.
+> Mais informações: <https://help.gnome.org/users/gnome-terminal/stable/>.
+
+- Abrir uma nova janela de terminal GNOME:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/gnome-terminal.md:6]: For Romance languages, the present indicative fits better.
```suggestion
- Abre uma nova janela de terminal GNOME:
```

File: pages.pt_BR/linux/gnome-terminal.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`gnome-terminal`
+
+- Executar um comando específico em uma nova janela de terminal:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/gnome-terminal.md:10]:
```suggestion
- Executa um comando específico em uma nova janela de terminal:
```

File: pages.pt_BR/linux/gnome-terminal.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`gnome-terminal -- {{commando}}`
+
+- Abrir uma nova aba na última janela aberta:
+
+`gnome-terminal --tab`
+
+- Definir o título da nova aba:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/gnome-terminal.md:18]:
```suggestion
- Define o título da nova aba:
```

File: pages.pt_BR/linux/gnome-terminal.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Abrir uma nova janela de terminal GNOME:
+
+`gnome-terminal`
+
+- Executar um comando específico em uma nova janela de terminal:
+
+`gnome-terminal -- {{commando}}`
+
+- Abrir uma nova aba na última janela aberta:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/gnome-terminal.md:14]:
```suggestion
- Abre uma nova aba na última janela aberta:
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

> Yay, It's my 500th PR review in tldr 🎉

**Pull request merged**